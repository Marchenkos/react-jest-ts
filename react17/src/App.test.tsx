import { shallow, ShallowWrapper } from "enzyme";

import App, { EditComponent } from './App';

describe('AccountDetailsScreen', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
      wrapper = shallow(<App />);
  });

  it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot();
  });

  it('should render an EditComponet', () => {
    expect(wrapper.find(EditComponent)).toBeTruthy();
  });
});
