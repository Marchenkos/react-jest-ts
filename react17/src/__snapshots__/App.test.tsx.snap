// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountDetailsScreen should match the snapshot 1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should match the snapshot 2`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render a EditCo the snapshot 1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render a EditComponet the snapshot 1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render an EditComponet  1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render an EditComponet 1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render an EditComponet the snapshot 1`] = `ShallowWrapper {}`;

exports[`AccountDetailsScreen should render an e the snapshot 1`] = `ShallowWrapper {}`;
